### DISCUSSION TITLE
Translations not approved

### DISCUSSION THREAD

**Discussion by joannavarrete** [codewars/codewars.com#2886] [General]
I've done a couple translations that doesn't seem to be noticed by anyone

https://www.codewars.com/kumite/637d03e8462774001ff1f4b7?sel=637d03e8462774001ff1f4b7
https://www.codewars.com/kumite/637e3e59ef7bb8a35e16a2a8?sel=637e3e59ef7bb8a35e16a2a8

They're my first translations, I don't know if I'm expected to do something else with them.

Could someone give them an eye and, in case they're correct, approve them. Else if something is missing I'll correct it.

I'd like to continue translating some more but I don't know if it's worth it if no one is reviewing them. 

Thanks a lot.

**Comment by JohanWiltink** [comment]
Have you tried Discord `#scala` ( if it exists ) or `#reviewing` ?

**Reply by joannavarrete** [reply]
I didn't know Discord. I've post a message in both servers.

Thank you.
